tlslite/tlsrecordlayer.py

```diff
@@ -722,6 +722,19 @@ def _getMsg(self, expectedType, secondaryType=None, constructorType=None):
                                 yield result
                             continue
 
+                    # If we received a hearbeat request ...
+                    if recordHeader.type == ContentType.heartbeat:
```

Comment:
where I can get here, if it is supported? I looked to some variable, but didnt find any, that got informations about negotiated parameters for connection